### DISCUSSION TITLE
Segurança em APIs ASP.NET com JWT e Bearer Authentication

### DISCUSSION THREAD

**Discussion by aps180180** [balta-io/forum#393] [Tira Dúvidas] [answered]
No meu projeto na hora de criar as migrations, se eu não tiver a conectionstring no appsettings, nao funciona,
apesar de ter as configurações no appsettings.development.
Alguém passou por isso?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @aps180180, como vai?
Eu explico este comportamento no curso de Fundamentos do ASP.NET, onde mostro em detalhes como trabalhar com configurações.
